app/cdap/text/text-en.yaml

```diff
@@ -102,25 +102,25 @@ features:
     copyright:
       firstLine: Copyright © 2014-2017 Cask Data, Inc.
       secondLine:
-        view: "View "
+        view: 'View '
```

Comment:
Please do not make unrelated changes to this file

Reply:
Done.